### PR TITLE
NXP-25188: remove browsersync and rely on polyserve

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -165,36 +160,18 @@ gulp.task('build', ['clean'], function(cb) {
...
-      }
-    },
-    // Run as an https by uncommenting 'https: true'
-    // Note: this uses an unsigned certificate which on first access
-    //       will present a certificate warning in the browser.
-    // https: true,
-    server: {
-      baseDir: ['.tmp', '.'],
-      middleware: [require('proxy-middleware')(proxyOptions)]
+    proxy: {
```
Comment (Gabez0r) [inline gulpfile.js]: need `hostname: '0.0.0.0',` in the options, in order for it to be accessible from outside

Comment (nelsonsilva) [reply gulpfile.js]: 👍 guess it should be optional but we can probably have it by default for convenience

**Comment by Gabez0r** [comment]
👍

**Comment by nmpcunha** [comment]
👍

**Pull request merged**